CMakeLists.txt

```diff
@@ -4,8 +4,7 @@
 
 cmake_minimum_required(VERSION 2.8)
 
-SET(PROJECTNAME wholeBodyModel)
-PROJECT(${PROJECTNAME})
+PROJECT(wholeBodyModel)
```

Comment:
Ah yes I understand. Then Im cool with the PR.


On Tue, Nov 3, 2015 at 11:55 PM, Silvio Traversaro <notifications@github.com
> wrote:

> In CMakeLists.txt
> <https://github.com/robotology/mex-wholebodymodel/pull/53#discussion_r43819619>
> :
>
> > @@ -4,8 +4,7 @@
> >
> >  cmake_minimum_required(VERSION 2.8)
> >
> > -SET(PROJECTNAME wholeBodyModel)
> > -PROJECT(${PROJECTNAME})
> > +PROJECT(wholeBodyModel)
>
> The PROJECT_NAME variable is used in the cmake to name the mex file. If
> we want to call it mexWholeBodyModel, we will have to change also all the
> matlab files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robotology/mex-wholebodymodel/pull/53/files#r43819619>
> .
>